Source/Armchair.swift

```diff
@@ -1703,23 +1737,19 @@ open class Manager : ArmchairManager {
     #endif
     
     private func hideRatingAlert() {
+        #if os(OSX)
```

Comment:
Why was all this edited? It still needs to be split by iOS and OSX to properly handle Mac App implementations.